common/src/main/resources/languages/en-gb.yml

```diff
@@ -43,4 +43,5 @@ error_reply_none_online: '[Error:](#ff3300) [Nobody in the last group of people
 error_last_message_not_group: '[Error:](#ff3300) [The last message you received was not a group message.](#ff7e5e)'
 error_no_messages_opt_out: '[Error:](#ff3300) [You have not yet sent or received group messages.](#ff7e5e)'
 removed_from_group_message: '[Removed you from the group message between:](#00fb9a) %1%'
-list_conjunction: 'and'
\ No newline at end of file
+list_conjunction: 'and'
+passthrough_shortcut_command_error: '[Error:](#ff3300) [Sending messages to passthrough channels using shortcut commands is not supported. Please switch to the channel first.](#ff7e5e)'
```

Comment:
I was trying to think of a better wording for this locale, but I don't have one. Worth mentioning that chat spoofing does work in Velocity still

Reply:
Yeah, I can't think of better wording for this either.